### PR TITLE
CY-1917: day2 operations for brokers and db nodes

### PR DESCRIPTION
**Pull request by tyacbovi**

### CODE REVIEW

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Cloudify HA Day-2 Operations
+description: Guides for manging Cloudify HA clusters
+weight: 80
+alwaysopen: false
+---
+
+# Day-2 operations for Cloudify HA
+
+## Introduction
```
Comment (ofercloudify) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: No point in this bullet if we dont have anything under it...
I suggest removing it.

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: Cloudify HA Day-2 Operations
+description: Guides for manging Cloudify HA clusters
+weight: 80
+alwaysopen: false
+---
+
+# Day-2 operations for Cloudify HA
```
Comment (ofercloudify) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: Cloudify cluster day 2 operations

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Introduction
+
+## Managing message queue cluster
```
Comment (ofercloudify) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: Message queue cluster operations

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+   cfy_manager brokers list
+``` 
+
+* An existing broker node can be removed from the cluster by running the following command:
+
+```bash  
+   cfy_manager brokers remove <nodename>
+``` 
+
+## Managing database cluster
```
Comment (ofercloudify) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: Database cluster operations

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md (hunk 2 above)
Comment (ofercloudify) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: What about manager cluster day 2 operations? adding/removing a node etc?

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+* For adding a management service cluster node, please install the new node on the same network settings as in Cloudify management service nodes.
+
+## Message queue cluster operations
+
+* A new broker node (an installed node) can be added to the cluster by running the following:
+
+  * Run the following command to connect the broker to the cluster:
+  
+    ```bash  
+       cfy_manager brokers add -j\--join-node <A resolvable hostname which complies to the format rabbit@<hostname> >
```
Comment (geokala) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: I'm fairly sure you don't need to provide the rabbit@ prefix, that'll be added automatically if there's no @ in the hostname.

Comment (geokala) [reply content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: However, it does need to target an existing member of the cluster (if memory serves- the help doc should confirm)

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+      cfy cluster brokers add <new broker name> <new broker address>
+    ```
+
+* Listing all the current message queue brokers in the cluster is retrieved by running the following command: 
+
+    ```bash  
+       cfy_manager brokers list
+    ``` 
+
+* An existing broker node can be removed from the cluster by running the following command:
```
Comment (geokala) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: First the broker to be removed should be uninstalled.

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+  * Run the following command to remove the broker from the cluster:
+    
+    ```bash  
+       cfy_manager brokers remove -r\--remove-node <nodename>
```
Comment (geokala) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: Secondly, this is run on one of the remaining cluster members.

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md (hunk 6 above)
Comment (geokala) [reply content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: (and it's not safe to remove a node when there are only two remaining- another node should be added first)

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
+---
+title: Cloudify cluster day 2 operations
+description: Guides for manging Cloudify HA clusters
+weight: 80
+alwaysopen: false
+---
+
+## Introduction
+
+Cloudify HA solution can contains three different clusters of database nodes, message queue brokers and Cloudify Managers.
```
Comment (geokala) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md]: All day 2 operations should only be conducted while the cluster is in maintenance mode.

**Pull request merged**